GenumKit/Stencil/Contexts.swift

```diff
@@ -165,29 +166,43 @@ extension StoryboardParser {
  - `structuredStrings`: `Dictionary` - contains strings structured by keys separated by '.' syntax
 */
 extension StringsFileParser {
-  public func stencilContext(enumName enumName: String = "L10n", tableName: String = "Localizable") -> Context {
+  public func stencilContext(enumName: String = "L10n", tableName: String = "Localizable") -> Context {
 
-    let entryToStringMapper = { (entry: Entry, keyPath: [String]) -> [String: AnyObject] in
+    let entryToStringMapper = { (entry: Entry, keyPath: [String]) -> [String: Any] in
       var keyStructure = entry.keyStructure
       Array(0..<keyPath.count).forEach { _ in keyStructure.removeFirst() }
-      let keytail = keyStructure.joinWithSeparator(".")
+      let keytail = keyStructure.joined(separator: ".")
 
       if entry.types.count > 0 {
-        let params = [
+        let params: [String : Any] = [
           "count": entry.types.count,
           "types": entry.types.map { $0.rawValue },
           "declarations": entry.types.indices.map { "let p\($0)" },
           "names": entry.types.indices.map { "p\($0)" },
-          "typednames": entry.types.enumerate().map { "p\($0): \($1.rawValue)" }
+          "typednames": entry.types.enumerated().map { "p\($0): \($1.rawValue)" }
+        ]
+        return ["key": entry.key,
+                "translation": entry.translation.newlineEscaped,
+                "params": params,
+                "keytail": keytail
         ]
-        return ["key": entry.key, "translation": entry.translation.newlineEscaped, "params": params, "keytail": keytail]
       } else {
-        return ["key": entry.key, "translation": entry.translation.newlineEscaped, "keytail": keytail]
+        return ["key": entry.key,
+                "translation": entry.translation.newlineEscaped,
+                "keytail": keytail
+        ]
       }
     }
 
-    let strings = entries.sort { $0.key < $1.key }.map { entryToStringMapper($0, []) }
-    let structuredStrings = structure(entries, mapper: entryToStringMapper, currentLevel: 0, maxLevel: 5)
+    let strings = entries
+        .sorted { $0.key.lowercased() < $1.key.lowercased() }
```

Comment:
Might be better to use str1.caseInsensitiveCompare(str2) here. Or anything that would ensure that any string case but also any diacritics and other niceties of Unicode are insensitively-compared properly (even if those are very rare to uncommon in keys)